dotnet/src/Functions/Functions.UnitTests/OpenAPI/Builders/Serialization/ArrayParameterSerializerTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json.Nodes;
+using Microsoft.SemanticKernel.Functions.OpenAPI.Builders.Serialization;
+using Xunit;
+
+namespace SemanticKernel.Functions.UnitTests.OpenAPI.Builders.Serialization;
+public class ArrayParameterSerializerTests
+{
+    [Fact]
+    public void ItShouldCreateParameterPerArrayItem()
+    {
+        // Arrange
+        var array = new JsonArray(1, 2, 3);
```

Comment:
Thought for a follow up PR, would be good to add some negative/edge test cases e.g. 

1. Arrays with duplicate values
2. Array value is the delimiter
3. Empty array

Reply:
Thanks Mark, will do it in the following PRs.